### DISCUSSION TITLE
System OS Packages

### DISCUSSION THREAD

**Discussion by trymeouteh** [rolldown/rolldown#1020] [Ideas]
Please release Rolldown the following ways for each OS.

- Windows
   - Winget
   - Scoop
- MacOS
   - Homebrew
 - Linux
   - Debian/Ubuntu Repo
   - Arch Repo
   - Nix

**Comment by jalmmor** [comment]
The ability to install Rolldown as a standalone application, without requiring Node.js—similar to how esbuild operates—is a fantastic idea, especially for users who don’t need additional add-ons

**Comment by IWANABETHATGUY** [comment]
Glad to see people who don't use Nodejs want to try `rolldown`, but distributing such binary at least means we need to maintain a another native cli, this would increase our maintenance burden, since we don't have enough bandwidth so we are not going to support this for now.
